src/components/listitems/ListItemRadioWithAmount.tsx

```diff
@@ -40,8 +40,8 @@ export const ListItemRadioWithAmount = (
 
   return (
     <PressableListItemBase onPress={pressHandler}>
-      <View>
-        <LabelSmallAlt color={theme["textBody-default"]}>
+      <View style={IOStyles.flex}>
```

Comment:
In this case, `flexShrink` is enough. You don't need to use the nuclear option (`flex: 1`): 
```suggestion
      <View style={{ flexShrink: 1 }}>
```
It's also better to add a `HSpacer` in the right `View` (just before `H6`) to add a safe margin between the two blocks

Reply:
Addressed into Addressed into bc0618b324d16a9a7098c9df9195467129d5eb4a